### PR TITLE
Getting Started: Sessions

### PR DESCRIPTION
**Pull request by bshaffer**
Dependent on https://github.com/googleapis/google-cloud-php/pull/2258

### CODE REVIEW

File: sessions/composer.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "require": {
+        "google/cloud-firestore": "^1.7"
```
Comment (bshaffer) [inline sessions/composer.json]: This version will need to be bumped once https://github.com/googleapis/google-cloud-php/pull/2258 is merged

**Comment by bshaffer** [comment]
The session handler  is merged! Waiting on a release from [`google-cloud-php`](https://github.com/googleapis/google-cloud-php/pull/2339) to merge this now.

File: sessions/php.ini [hunk 2]
```diff
@@ -0,0 +1,2 @@
+; enable the gRPC extension
+extension=grpc.so
```
Comment (dwsupplee) [inline sessions/php.ini]: Should we include protobuf as well?

File: sessions/test/DeployTest.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\GettingStarted\Sessions;
+
+use Google\Cloud\TestUtils\AppEngineDeploymentTrait;
+use Google\Cloud\TestUtils\TestTrait;
```
Comment (dwsupplee) [inline sessions/test/DeployTest.php]: It looks like we may need to require the `TestUtils` package.

File: sessions/phpunit.xml.dist [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<phpunit bootstrap="vendor/autoload.php">
+    <testsuites>
+        <testsuite>
+            <directory>test</directory>
+        </testsuite>
+    </testsuites>
+    <filter>
```
Comment (dwsupplee) [inline sessions/phpunit.xml.dist]: Is this needed?

File: sessions/phpunit.xml.dist [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<phpunit bootstrap="vendor/autoload.php">
+    <testsuites>
+        <testsuite>
```
Comment (dwsupplee) [inline sessions/phpunit.xml.dist]: I'm getting the following error when running `phpunit`:

```
  Line 19:
  - Element 'testsuite': The attribute 'name' is required but missing.
```

File: sessions/.gitignore [hunk 6]
```diff
@@ -0,0 +1 @@
+/vendor
```
Comment (dwsupplee) [inline sessions/.gitignore]: The gitignores for the other samples include `/.idea` and `.env`, for consistency should we include those here too?

File: sessions/test/DeployTest.php (hunk 3 above)
Comment (bshaffer) [reply sessions/test/DeployTest.php]: This is added as a global composer dep in `.travis.yaml`.

File: sessions/phpunit.xml.dist (hunk 4 above)
Comment (bshaffer) [reply sessions/phpunit.xml.dist]: nope!

File: sessions/phpunit.xml.dist (hunk 5 above)
Comment (bshaffer) [reply sessions/phpunit.xml.dist]: strange, the name attribute is there

File: sessions/.gitignore (hunk 6 above)
Comment (bshaffer) [reply sessions/.gitignore]: Yes, I think these should be included for all laravel/lumen samples!

Comment (bshaffer) [reply sessions/.gitignore]: The reason I didn't include these is that this is not a Laravel/Lumen sample. So at least `.env` is not necessary. `/.idea` I could go either way on, but I'll add it!

File: sessions/phpunit.xml.dist (hunk 5 above)
Comment (bshaffer) [reply sessions/phpunit.xml.dist]: NM, I must have been looking at the wrong file

File: sessions/README.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+# Session handing with PHP on Google Cloud Platform
```
Comment (dwsupplee) [inline sessions/README.md]:
```suggestion
# Session handling with PHP on Google Cloud Platform
```

**Pull request merged**